assets/src/design-system/components/tooltip/index.js

```diff
@@ -48,7 +48,7 @@ const TooltipContainer = styled.div`
   align-items: center;
   text-align: center;
   flex-direction: row;
-  max-width: 13em;
+  max-width: 14em;
```

Comment:
designs showed a [wider snackbar](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=6310%3A569093)

Reply:
pretty sure what you updated is still relevant but make sure you're using the 1.7+ file for reference. https://www.figma.com/file/1YM3jA9h4QGc4xLdyaXIJS/Stories-Sprint-1.7?node-id=7347%3A56346 

I updated all the tickets to reference this, i think. But this epic was made before the designs were transferred to this file so I want to make sure we're careful for discrepancies that are under the radar.